data/map planets.txt

```diff
@@ -2098,6 +2104,13 @@ planet Hestia
 		threshold 8500
 		fleet "Large Republic" 18
 
+planet Hevring
+	attributes moon paradise uninhabited
+	landscape land/moon2
+	description `Hevring, moon of the gas giant AEgir, was once an extrasolar mining colony from a corporation of old Earth, established in the first hyperdrive-fuelled expansion of humanity amongst the stars.`
```

Comment:
```suggestion
	description `A moon of the gas giant Aegir, Hevring was once an extrasolar mining colony from a corporation of old Earth, established in the first hyperdrive-fueled expansion of humanity amongst the stars.`
```

Reply:
The gas giant's name is acrually spelt AEgir. Everything else looks alright.

Reply:
Or what we could do is leave it as Aegir and then add the Œ when we get unicode support.

Reply:
https://en.wikipedia.org/wiki/Epsilon_Eridani_b – AEgir

Reply:
> Or what we could do is leave it as Aegir and then add the Œ when we get unicode support, since tonnes of people are probably going to complain about the AE.

Yeah that's what I was thinking when I made the suggestion. Up to you.